drivers/SmartThings/zigbee-switch/src/inovelli-vzm32-sn/init.lua

```diff
@@ -0,0 +1,520 @@
+    device:send(clusters.OnOff.server.commands.Off(device))
+  else
+    device:emit_event(capabilities.switch.switch("off"))
+    local dev = device:get_parent_device()
+    local send_configuration = function()
+      dev:send(cluster_base.build_manufacturer_specific_command(
+            dev,
+            PRIVATE_CLUSTER_ID,
+            PRIVATE_CMD_NOTIF_ID,
+            MFG_CODE,
+            utils.serialize_int(0,4,false,false)))
+    end
+    device.thread:call_with_delay(1,send_configuration)
+  end
+end
+
+local function switch_level_handler(driver, device, command)
+  if device.network_type ~= st_device.NETWORK_TYPE_CHILD then
+    device:send(clusters.Level.server.commands.MoveToLevelWithOnOff(device, math.floor(command.args.level/100.0 * 254), command.args.rate or 0xFFFF))
+  else
+    device:emit_event(capabilities.switchLevel.level(command.args.level))
+    device:emit_event(capabilities.switch.switch(command.args.level ~= 0 and "on" or "off"))
+    local dev = device:get_parent_device()
+    local send_configuration = function()
+      dev:send(cluster_base.build_manufacturer_specific_command(
+            dev,
+            PRIVATE_CLUSTER_ID,
+            PRIVATE_CMD_NOTIF_ID,
+            MFG_CODE,
+            utils.serialize_int(getNotificationValue(device),4,false,false)))
+    end
+    device.thread:call_with_delay(1,send_configuration)
+  end
+end
+
+local function set_color_temperature(driver, device, command)
+  device:emit_event(capabilities.colorControl.hue(100))
+  device:emit_event(capabilities.colorTemperature.colorTemperature(command.args.temperature))
+  device:emit_event(capabilities.switch.switch("on"))
+  local dev = device:get_parent_device()
+  local send_configuration = function()
+    dev:send(cluster_base.build_manufacturer_specific_command(
+          dev,
+          PRIVATE_CLUSTER_ID,
+          PRIVATE_CMD_NOTIF_ID,
+          MFG_CODE,
+          utils.serialize_int(getNotificationValue(device, 100),4,false,false)))
+  end
+  device.thread:call_with_delay(1,send_configuration)
+end
+
+local function set_color(driver, device, command)
+  device:emit_event(capabilities.colorControl.hue(command.args.color.hue))
+  device:emit_event(capabilities.colorControl.saturation(command.args.color.saturation))
+  device:emit_event(capabilities.switch.switch("on"))
+  local dev = device:get_parent_device()
+  local send_configuration = function()
+    dev:send(cluster_base.build_manufacturer_specific_command(
+          dev,
+          PRIVATE_CLUSTER_ID,
+          PRIVATE_CMD_NOTIF_ID,
+          MFG_CODE,
+          utils.serialize_int(getNotificationValue(device),4,false,false)))
+  end
+  device.thread:call_with_delay(1,send_configuration)
+end
+
+local function occupancy_attr_handler(driver, device, occupancy, zb_rx)
+  device:emit_event(occupancy.value == 0x01 and capabilities.motionSensor.motion.active() or capabilities.motionSensor.motion.inactive())
+end
```

Comment:
this should also be handled by the defaults

Reply:
My mistake. We do only have default handling for the occupancy attribute for the occupancy sensor capability. Our motion sensor defaults use the IAS Zone Status attribute.